vmck/api.py

```diff
@@ -23,6 +29,28 @@ def home(request):
     })
 
 
+def create_token(request):
+    auth_header = request.headers['Authorization']
+    token_type, _, credentials = auth_header.partition(' ')
```

Comment:
Please add parantheses around the variables, it makes it more clear that you're unpacking a list or tuple.

Reply:
See django's api on how to verify user credentials: https://docs.djangoproject.com/en/2.2/topics/auth/default/#authenticating-users